pkg/crypto/crypto.go

```diff
@@ -61,17 +64,33 @@ func encrypt(plaintext string, key []byte) (string, error) {
 	writer.Close()
 	encoder.Close()
 
-	return string(msgBuffer.Bytes()), nil
+	// Remove unnecessary characters and trim new lines before storing as
+	// part of node Annotations username.
+	encryptedStr := string(msgBuffer.Bytes())
+	encryptedStr = strings.Replace(encryptedStr, startTag, "", -1)
+	encryptedStr = strings.Replace(encryptedStr, endTag, "", -1)
+	encryptedStr = strings.Trim(encryptedStr, "\n")
+	return encryptedStr, nil
 }
 
 // decrypt converts encrypted contents to plaintext using the given key as a passphrase
 func decrypt(encrypted string, key []byte) (string, error) {
 	if key == nil {
 		return "", errors.New("decryption passphrase cannot be empty")
 	}
+	// As we trim and remove unnecessary characters during encrypt call, we need to
+	// insert them back to encrypted string to get original value.
+	// 1. add start tag 2. add new lines 3.encrypted string 4. add new line 5. add end tag
+    var strBld strings.Builder
+	strBld.WriteString(startTag)
+	strBld.WriteString(newLine)
+	strBld.WriteString(newLine)
+	strBld.WriteString(encrypted)
+	strBld.WriteString(newLine)
+	strBld.WriteString(endTag)
```

Comment:
I'd prefer avoiding use this pattern as I think it is unnecessarily verbose.
Could do this instead:
```
encrypted = fmt.Sprintf("%s\n\n%s\n%s", startTag, encrypted, endTag)
```